### PR TITLE
[Input] Change default svg to component - Android

### PR DESCRIPTION
**Pull request by ayush547**
### Platforms Impacted
- [ ] iOS
- [ ] macOS
- [ ] win32 (Office)
- [ ] windows
- [x] android

### Description of changes

Using svg as a file poses some issues while importing and requires some setup. Changing svg into a component instead helps remove these issues.

### Verification

(how the change was tested, including both manual and automated tests)

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Screenshot or description before this change | Screenshot or description with this change |

### Pull request checklist

This PR has considered (when applicable):
- [x] Automated Tests
- [ ] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

**Review by Saadnajmi — APPROVED** [review]

File: packages/components/Input/src/dismissSvg.tsx
Comment (rohanprasadofficial) [inline packages/components/Input/src/dismissSvg.tsx:1]: Putting this file inside a 'assets' folder inside src only would be good.

**Review by rohanprasadofficial — APPROVED** [review]
Putting this file inside a 'assets' folder.

File: packages/components/Input/package.json [hunk 1]
```diff
@@ -40,11 +40,13 @@
     "@fluentui-react-native/test-tools": ">=0.1.1 <1.0.0",
     "@fluentui-react-native/scripts": "^0.1.1",
     "react": "17.0.2",
-    "react-native": "^0.68.0"
+    "react-native": "^0.68.0",
+    "react-native-svg": "^12.3.0"
```
Comment (ankraj12) [inline packages/components/Input/package.json:44]: Should this be specified in dependencies instead?

**Comment by ankraj12** [comment]
Can you share why kind of issues were observed in current code..

File: packages/components/Input/src/dismissSvg.tsx
Comment (ayush547) [reply packages/components/Input/src/dismissSvg.tsx:1]: Done

**Comment by ayush547** [comment]
> Can you share what kind of issues were observed in current code.

SVGs require some [transformer](https://yarnpkg.com/package/react-native-svg-transformer) to be recognized as a module. Converting SVGs to component allows direct import. This should be better for partner consumption.

File: packages/components/Input/package.json (hunk 1 above)
Comment (ayush547) [reply packages/components/Input/package.json:44]: Intentional

**Pull request merged**